### ISSUE TITLE
a11y for Breadcrumb

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-ngx#10085]
#### Is this a bug, enhancement, or feature request?
a11y related bug/enhancement
#### Briefly describe your proposal.
Currently to navigate the elements of the Breadcrumb the user needs to use the Tab key. The elements should be navigated with left and right arrow keys.
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
latest
